src/domain/event/__tests__/EventPage.test.tsx

```diff
@@ -636,4 +636,56 @@ describe('refetch of event works correctly', () => {
       ).toBeInTheDocument();
     });
   });
+
+  it('renders a link which leads to all events of the organisation', async () => {
+    renderComponent();
+    await waitForRequestsToComplete();
+    expect(
```

Comment:
queryBy should only be used when something is not in the document:  https://testing-library.com/docs/queries/about/
>queryBy...: Returns the first matching node for a query, and return null if no elements match. This is useful for asserting an element that is not present. Throws an error if more than one match is found (use queryAllBy instead if this is OK).
>findBy...: Returns a Promise which resolves when an element is found which matches the given query. The promise is rejected if no element is found or if more than one element is found after a default timeout of 1000ms. If you need to find more than one element, use findAllBy.

This could be more preferred check:
`await screen.findByRole('link', {  name: /näytä järjestäjän tapahtumat/i });`